app/services/csv_to_json_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Convert CSV to JSON for virtual objects
+class CsvToJsonService
```

Comment:
I'd like to see this have a less generic name. add something about virtual merge in there.

Reply:
Coming.

Reply:
Done.